.gitlab/deps_build.yml

```diff
@@ -40,15 +42,16 @@ build_libbcc_x64:
 
 build_libbcc_arm64:
   extends: .build_libbcc_common
+  rules:
+    - <<: *if_run_all_builds
```

Comment:
Do these need the `if_run_all_builds` since they are manual jobs?

Reply:
We can probably leave them here indeed.